pwiz/data/vendor_readers/Waters/ChromatogramList_Waters.cpp

```diff
@@ -329,10 +355,27 @@ PWIZ_API_DECL void ChromatogramList_Waters::createIndex() const
         }
     }
 
+    long analogChannels = rawdata_ -> AnalogIntensitiesByChannel().size();
+
+    for(int ch=0; ch < analogChannels; ch++)
+    {
+        index_.push_back(IndexEntry());
+        IndexEntry& ie = index_.back();
+        ie.index = index_.size() - 1;
+        ie.function = -1;
+        ie.offset = ch;
+        ie.Q1 = 0;
+
+        std::string temp = rawdata_->AnalogChannelNames()[ch];
+
+        boost::algorithm::trim(temp);
+        ie.id = temp;
+        ie.chromatogramType = MS_chromatogram;

```

Comment:
I'm still not sure what kind of chromatograms these are. We also should (bordering on must) have an example file to include in Reader_Waters_Test.data to test this code.

Reply:
I´m on it.